JitsiConference.js

```diff
@@ -1273,9 +1273,11 @@ function setupListeners(conference) {
                     session, conference.settings);
             });
 
-        conference.room.addListener(XMPPEvents.CONFERENCE_SETUP_FAILED,
-            function () {
-                conference.statistics.sendSetupFailedEvent();
+        conference.room.addListener(XMPPEvents.CONNECTION_ICE_FAILED,
```

Comment:
Yes, this is intentional. The idea is that we are getting rid of setupFailedEvent that we used to use for callstats and is fired every time on CONFERENCE_SETUP_FAILED event. I've checked and we fire this event(CONFERENCE_SETUP_FAILED) on error in oniceconnectionstatechange, createAnswer, setLocalDescription, setRemoteDescription. And in all cases there are corresponding callstats errors we log except oniceconnectionstatechange. And as callstats.io had introduced new error type called iceConnectionFailure, I've made new event XMPPEvents.CONNECTION_ICE_FAILED and when this happen we log that error and fire CONFERENCE_SETUP_FAILED as we used to before my modifications.